spec/models/provider_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Provider, type: :model do
+  let(:subject) { create(:provider) }
+  describe 'it has all the attributes' do
+    it "should include the :id attribute" do
+      expect(subject.attributes).to include("id")
+    end
+    it "should include the :name attribute" do
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] RSpec/ExampleWording: Do not use should when describing your tests.